ujson.go

```diff
@@ -6,17 +6,21 @@ import (
 )
 
 type JSON struct {
-	root interface{}
+	Root interface{}
```

Comment:
what's the motivation for exposing this?  I would like to try to avoid leaking internals, should we just add an accessor method?